packages/react/src/oidc/User.ts

```diff
@@ -9,10 +9,42 @@ export enum OidcUserStatus {
 }
 
 export type OidcUser = {
-    user: any,
+    user: OidcUserInfo,
     status: OidcUserStatus
 }
 
+export interface OidcUserInfo {
```

Comment:
Hi @zyofeng Thank you for your awesome PullRequest.

Is it possible to be able to inject our custom Type ?
Is it possible to be able to make OidcUserInfo extensible (for exemple to add new properties) because each oidc provider can add custom properties ?

Reply:
i want to mock useOidcUser but i gett error for oidcUser: OidcUserInfo

import {  useOidcUser, useOidc, OidcUserStatus } from "@axa-fr/react-oidc";



jest.mock("@axa-fr/react-oidc");

const useOidcMock = useOidc as jest.MockedFunction<typeof useOidc>;

const useOidcUserMock = useOidcUser as jest.MockedFunction<typeof useOidcUser>;

it("renders correctly", () => {
  useOidcUserMock.mockImplementation(({ oidcUser:{sub:"fgfhfghghj"}, oidcUserLoadingState: OidcUserStatus.Loaded }));
  useOidcMock.mockImplementation(({
    login: async () => { },
    logout: async () => { },
    renewTokens: async () => { },
    isAuthenticated: true,
  }));
  const tree = renderer
    .create(
      < BrowserRouter >
        <AccountPopover />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});


I open issue for this in  #926